Cargo.toml

```diff
@@ -33,7 +33,7 @@ once_cell = "1"
 url = "2"
 socks = { version = "0.3", optional = true }
 serde = { version = "1", optional = true }
-serde_json = { version = "1", optional = true }
+serde_json = { version = "=1.0.93", optional = true }
```

Comment:
Wonder if this could be a less than range to not force users to a specific version?